x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-control-numbers.adoc

```diff
@@ -0,0 +1,28 @@
+= X12 EDI Connector 2.5 - Control Numbers - Mule 4
```

Comment:
2.5 instead of 2.2? This is a separate EDI connector?
Just making sure....  ;-)

Reply:
Yep! This is the X12 connector - it is essentially the same thing, but for a different connector on a different version. Oy!